src/mmrelay/meshtastic_utils.py

```
@@ -3009,61 +3096,33 @@ async def check_connection() -> None:
-            if connection_type == CONNECTION_TYPE_BLE:
-                if not ble_skip_logged:
-                    logger.info(
-                        "BLE connection uses real-time disconnection detection - health checks disabled"
+            try:
+                loop = asyncio.get_running_loop()
+                await asyncio.wait_for(
+                    loop.run_in_executor(None, meshtastic_client.getMyNodeInfo),
+                    timeout=DEFAULT_MESHTASTIC_OPERATION_TIMEOUT,
+                )
+
+            except Exception as e:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)